tokens.xml

```diff
@@ -3091,6 +3135,21 @@ This creature's power and toughness are each equal to the number of instant and
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Faerie Spy Token</name>
+            <text>Flying, haste
+                Whenever this creature deals combat damage to a player, draw a card.</text>
```

Comment:
```suggestion
Whenever this creature deals combat damage to a player, draw a card.</text>
```
card text should not be indented, it gets copied over:
![image](https://user-images.githubusercontent.com/36401181/177195946-159f7fe7-aa44-40dd-897b-058bf2d51b8b.png)